src/content_scripts/applyColorSchemeJs.js

```diff
@@ -144,9 +146,21 @@ function _OffListener(func, mediaQueryList, isOnChange) {
     return hook;
 }
 
+/**
+ * Check if an object is a MediaQueryList
+ * The correctness of this relies on Firefox's XPCNativeWrapper
+ *
+ * @private
+ * @param {function} obj
+ * @returns {boolean}
+ */
+function _checkIsMediaQueryList(obj) {
```

Comment:
Please remove that `_` prefix from all functions, we don't use it here. It's enough to use use JSDoc to mark a function as `@private`.